View/ExpressionLanguage.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the EzCoreExtraBundle package.
+ *
+ * @copyright Jérôme Vieilledent <jerome@vieilledent.fr>
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace Lolautruche\EzCoreExtraBundle\View;
+
+use Symfony\Component\ExpressionLanguage\ExpressionLanguage as BaseExpressionLanguage;
+
+class ExpressionLanguage extends BaseExpressionLanguage
+{
+    public function __construct($cache = null, array $providers = array())
```

Comment:
you should type hint to `ParserCacheInterface`

Reply:
Nope, this is deprecated as of Symfony 3.2. It has to be replaced by `CacheItemPoolInterface`, but I don't want to enforce it as it might break.